frameworks/cassandra/tests/test_overlay.py

```diff
@@ -43,6 +45,7 @@ def teardown_module(module):
 @pytest.mark.sanity
 @pytest.mark.smoke
 @pytest.mark.overlay
+@pytest.mark.skipif('os.environ.get("SECURITY") == "strict"')
```

Comment:
I find the pytest docs somewhat useless, so was following the odd example from shakedown's use of skipif for version numbers.  I'm not sure if they have a need for it (does it possibly delay execution?)  I don't.